iree/compiler/Dialect/Flow/IR/FlowOpFolders.cpp

```diff
@@ -159,6 +176,19 @@ struct InsertImmutabilityPreservingStreamClones
         });
         didClone = true;
       }
+
+      // TODO(GH-5492): This is a temporary solution to address the issue
```

Comment:
nit: use TODO(#issue) so github will automatically link the issue number
```suggestion
      // TODO(#5492): This is a temporary solution to address the issue
```